_content/gateways/pycom/index.md

```diff
@@ -0,0 +1,35 @@
+# Pygate description
```

Comment:
Please follow the header format so this page gets rendered correctly.

For example:

```
---
title: Pycom Pygate
section: Hardware
redirect_from:
 - /pycom/
image: /gateways/pycom/image.jpg
zindex: 900
---
```

You may also need to change this header.